src/api/config.ts

```diff
@@ -1,7 +1,5 @@
-import Config from 'react-native-config';
```

Comment:
I doubt if we need this file at all. What benefits does it bring? Imho only increases complexity.

Reply:
I don't know either. Perhaps you're right